src/app/pages/main/detail/detail.component.html

```diff
@@ -11,6 +11,28 @@
     </div>
     <label nz-checkbox (click)="$event.stopPropagation()" [(ngModel)]="currentTodo.completedFlag"></label>
     <span>标记完成</span>
+    <nz-divider nzText="标签" nzOrientation="left"></nz-divider>
```

Comment:
Tag has no superior priority to title and date. It should be placed after date and before description.